automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterConcurrencySpec.scala

```diff
@@ -128,31 +76,34 @@ class ClusterConcurrencySpec extends FreeSpec with LeonardoTestUtils with Parall
     }
 
     // create -> wait -> stop -> wait -> delete
-    "should be able to delete a stopped cluster" in withWebDriver { implicit driver =>
+    "should be able to delete a stopped cluster" in {
       withProject { project => implicit token =>
         logger.info(s"${project.value}: should be able to delete a stopped cluster")
 
         withNewCluster(project) { cluster =>
-          // delete after stop is complete
-          stopAndMonitor(cluster.googleProject, cluster.clusterName)
+          withWebDriver { implicit driver =>
```

Comment:
I don't think this needs webDriver at all. I know `stopAndMonitor` and `startAndMonitor` current take an implicit webDriver, but I don't see how it's used. Can we remove webDriver from stop/start entirely?

Reply:
removed `implicit webDriver`